src/System.Private.CoreLib/src/System/TimeZoneInfo.cs

```diff
@@ -43,7 +43,7 @@ internal enum TimeZoneInfoOptions
 
 
     [Serializable]
-    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Core, Version=3.5.0.0, Culture=Neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
Nit: `Culture=Neutral` should have lower-case `neutral` - to be consistent with what we have everywhere else.